server/datastore/mysql/microsoft_mdm.go

```diff
@@ -160,10 +160,10 @@ func (ds *Datastore) MDMWindowsInsertCommandForHosts(ctx context.Context, hostUU
 func (ds *Datastore) mdmWindowsInsertHostCommandDB(ctx context.Context, tx sqlx.ExecerContext, hostUUID, commandUUID string) error {
 	stmt := `
 INSERT INTO windows_mdm_command_queue (enrollment_id, command_uuid)
-VALUES ((SELECT id FROM mdm_windows_enrollments WHERE host_uuid = ?), ?)
+VALUES ((SELECT id FROM mdm_windows_enrollments WHERE host_uuid = ? OR mdm_device_id = ?), ?)
```

Comment:
any thoughts on this? I went with this vs having a separate method just because it was easier and we're under the clock.

Reply:
I think that's ok but I'd rename the method and parameter to reflect that though? I don't think there's a risk of conflicting values (a host uuid cannot look like/be a valid device ID from what we've seen), so the only thing that could go wrong would maybe be performance if the OR causes index selection issues (something that we can address if/when it happens).

Reply:
sounds good! thanks for the reminder about the parameters 😅 . Fixed!